### PR TITLE
Game manager/reorganize methods

### PR DESCRIPTION
**Pull request by GregJMitchell**
This branch does:
-Moves all game stats methods into game manager

### CODE REVIEW

File: test/game_manager_test.rb [hunk 1]
```diff
@@ -2,4 +2,76 @@
...
+  end
+
+  def test_it_has_readable_attributes
+    game_manager = mock('game manager object')
+    game_1 = mock('game object 1')
+    game_2 = mock('game object 2')
+    game_3 = mock('game object 2')
+    game_4 = mock('game object 2')
+    game_5 = mock('game object 2')
+    game_manager.stubs(:games).returns([game_1, game_2, game_3, game_4, game_5])
```
Comment (s-steel) [inline test/game_manager_test.rb:29]: Nice use of mocks and stubs

File: test/game_manager_test.rb [hunk 2]
```diff
@@ -2,4 +2,76 @@
...
+  def test_it_can_find_percentage_visitor_wins
+    assert_equal 0.36, @game_manager.percentage_visitor_wins
+    assert_equal 178, @game_manager.percentage_visitor_win_helper
+  end
+
+  def test_it_can_find_percentage_ties
+    assert_equal 0.21, @game_manager.percentage_ties
+    assert_equal 105, @game_manager.percentage_ties_helper
+    assert_equal 0.43, @game_manager.percentage_home_wins
+    assert_equal 213, @game_manager.percentage_home_win_helper
```
Comment (s-steel) [inline test/game_manager_test.rb:50]: I like all the tests here.

**Review by s-steel — APPROVED** [review]
Looks good.  Way to use mocks and stubs and increase testing.

**Review by c-bartell — APPROVED** [review]

**Comment by cpfergus1** [comment]
I agree with the other's assessment. Thanks and good work

**Pull request merged**